### ISSUE TITLE
Init data while navigate with react-router

### ISSUE THREAD

**Issue by bsdo64** [goatslacker/alt#660]
I'm using react router, normalizr, immutable now.

When user navigate route, react router listener fire init action all store data from api server by ajax.

After ajax data call success, normalize to domain store data (POSTS, USERS, COMMENTS, CATEGORIES...) with normalizr and setState each domain data to domain store (which are listening init action)

But the problem is with domain stores and ui stores, these count more than 30~ and each containers listening the stores re-render too many more than 1sec.

Is it correct way init data with alt?? I'm not using isomorphic app. Just client app
